### PR TITLE
Add information about deprecation in Toolbox

### PR DESCRIPTION
**Pull request by olegchir**
This commits adds a new initial screen, explaining that Projector Launcher will eventually fade away from Toolbox.

It can be dismissed with the "don't show again" option that is stored in the preferences file in the home directory, it's the same place where the last URL is stored.

### CODE REVIEW

File: projector-launcher/build.gradle.kts [hunk 1]
```diff
@@ -188,3 +188,14 @@ tasks.create<Exec>("electronProductionRun") {
   workingDir(project.file(distDir))
   commandLine(npmCommand + listOf("run", "electron", "--", "."))
 }
+
+tasks.create<Exec>("electronRun") {
+  workingDir(project.file(distDir))
+  commandLine(npmCommand + listOf("run", "electron", "--", "."))
+}
+
+tasks.create<Exec>("electronBuildAndRun") {
```
Comment (SerVB) [inline projector-launcher/build.gradle.kts:197]: Could you please clarify why new tasks are needed? We have a similar task above, doesn't it work? Please update readme then

**Review by SerVB — APPROVED** [review]
Overall OK. Please also clarify the info I've asked for in the comment. Also, please update [the changelog](https://github.com/JetBrains/projector-client/blob/master/projector-launcher/CHANGELOG.md). Unfortunately, I'm unable to test it because I don't have a PC now. @Avvessalom , please take a look and test it: we need to publish this update asap. Requesting your review

**Comment by olegchir** [comment]
Anyway I'm still waiting for the real text from the MKT-20182

**Comment by olegchir** [comment]
The final version of the text is ready. All links checked, HTML markup fixed, everything seems fine.

@Avvessalom could you please run the release procedure?

File: projector-launcher/build.gradle.kts (hunk 1 above)
Comment (olegchir) [reply projector-launcher/build.gradle.kts:197]: Fixed in eb3bf403bcae6809e2c8ac319743fea0cc74b982

**Comment by zhipengzuo** [comment]
hello, can i ask why projecotor been removed from toolbox ? will jetbrains continue develop projecotor as before ?

**Comment by SerVB** [comment]
Hi @zhipengzuo , here is rationale (in Russian): https://twitter.com/olegchir/status/1481655271441059842.

TL;DR: we are continuing developing Projector, and removal of the app from Toolbox can be just temporary. Please use GitHub Releases until we restore downloading in Toolbox.

@olegchir, probably we should post the same on https://twitter.com/ProjectorJB in English.

File: projector-launcher/src/main/kotlin/ElectronApp.kt [hunk 2]
```diff
@@ -46,6 +49,7 @@ class ElectronApp(val url: String) {
   lateinit var mainWindowNextUrl: String
   var initialized = false
 
+
```
Comment (Avvessalom) [inline projector-launcher/src/main/kotlin/ElectronApp.kt:52]: If you decide to change something else, then please also remove this extra line

**Review by Avvessalom — APPROVED** [review]
LGTM! Suggested several cosmetic changes.

File: projector-launcher/build.gradle.kts [hunk 3]
```diff
@@ -188,3 +188,14 @@ tasks.create<Exec>("electronProductionRun") {
   workingDir(project.file(distDir))
   commandLine(npmCommand + listOf("run", "electron", "--", "."))
 }
+
+tasks.create<Exec>("electronRun") {
+  workingDir(project.file(distDir))
+  commandLine(npmCommand + listOf("run", "electron", "--", "."))
+}
+
```
[the remark this answers is not in the document]
Comment (olegchir) [reply projector-launcher/build.gradle.kts]: Wow, excellent idea, my friend! Created a new group "development" and now it's much easier to find these tasks in the Gradle plugin. Implemented in 7c08055.

File: projector-launcher/src/main/kotlin/ElectronApp.kt (hunk 2 above)
Comment (olegchir) [reply projector-launcher/src/main/kotlin/ElectronApp.kt]: Just performed a more extensive cleanup here: 01b2dcf.
Hope nothing important is removed lol
